src/lib/features/Footer.svelte

```diff
@@ -1,20 +1,39 @@
 <script lang="ts">
+  import { cn } from '../utils/utils';
   import { FrequencyLogo, XLogo, DiscordLogo } from '../assets';
   import IconButton from '../atoms/IconButton.svelte';
+
+  export let privacyHref: string = '/privacy';
+  export let type: 'light' | 'dark' = 'light';
+
+  let fgColor = {
+    dark: 'white',
+    light: 'black',
+  }[type];
 </script>
 
-<footer class="sm md:md flex flex-col items-center gap-f16 border-t border-black py-f48 md:gap-f24">
-  <div class="flex gap-f40">
-    <a href="/privacy" class="underline transition duration-[0.3s] hover:text-brightBlue">Privacy Policy</a>
-    <a href="/terms" class="underline transition duration-[0.3s] hover:text-brightBlue">Terms of Conditions</a>
-  </div>
+<footer
+  class={cn(
+    `sm md:md flex flex-col items-center gap-f16 border-t-[2px] border-current text-${fgColor} mx-auto my-0 max-w-screen-xl py-f48 md:gap-f48`
+  )}
```

Comment:
Watch out--building TW class names with dynamic strings (`text-${fgColor}`) can be problematic.
Also, if you're not merging/concatenating classes, I don't think you need the `cn(...)` method; you can just say `class="..."`.

Or does the cn(...) function somehow overcome the gotchas that dynamic TW class names presents?

Anyway, see PR #50 for an alternate approach to this styling issue.

Reply:
I think TW recommends something more like the following:
```svelte
<footer class="... { fgColor === 'light' ? 'text-black' : 'text-white'}"/>
```

Reply:
☝🏻 reason being that the TW parses is a very simple string-matcher on class names; it doesn't recognize `text-${fgColor}` as a TW class, but it does fine with `'text-white' : 'text-black'` (it compiles both styles)